src/test/run-fail/multi-panic.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright 2015 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// error-pattern:thread '<unnamed>' panicked at
+// error-pattern:note: Run with `RUST_BACKTRACE=1` for a backtrace.
+// error-pattern:thread '<main>' panicked at
+
+fn main() {
+    let _ = std::thread::spawn(|| {
+        panic!();
+    }).join();
+
+    panic!()
+}
```

Comment:
Could this test re-exec itself and check the output rather than relying on `output-pattern` (which isn't exhaustive)?

That way it can also make sure that there's exactly one instance of `RUST_BACKTRACE`